submissions/shramkoweb/js-dom/index.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
```

Comment:
I guess `content="width=device-width"` will be enough here. HTML validators sometimes give warnings about it

Reply:
Can I show when I have problem with this case ?